File: 1564.py
lista = []
s = 0
while True:
  try:
    N = int(input())
    if 0 <= N <= 100:
        if N == 0:
            lista += ['vai ter copa!']
        elif N > 0:
            lista += ['vai ter duas!']
    s += 1
  except EOFError:
    break
for d in range(0, s):
    print(lista[d])

File: 1564.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main(){
    int reclamacoes;
    while(cin >> reclamacoes){
        if(!reclamacoes)
            cout << "vai ter copa!\n";
        else
            cout << "vai ter duas!\n";
    }
    return 0;
}
